README.md

```diff
@@ -121,14 +121,17 @@ Next clone the CI script:
 
 Manually define the variables, Travis would otherwise define for you. These are required:
 
-    export TRAVIS_BRANCH=melodic-devel
+    export TRAVIS_BRANCH=melodic-devel   # Replace with the repository branch you intend to test against CI
```

Comment:
Being able to replace those definitions with suitable, situation-specific values is obvious from my point of view.
Maybe adding a single comment on top of this section suffices to this aim?